core/templates/core/post-list.html

```diff
@@ -7,34 +7,31 @@
 
 {% block content %}
 <div class="container">
-    <a href="{% url 'admin:core_post_add' %}" class="admin-link">Add New Post</a>
-    <p>
-    </p>
-    <ul class="post-list">
+    <div class="mx-auto col-10 col-md-8 col-lg-8 mt-lg-3">
+        <a href="{% url 'admin:core_post_add' %}" class="admin-link mb-3">Add New Post</a>
         {% for post in posts %}
-        <li>
-            <div class="card bg-dark shadow-sm mb-2 text-light"
-            >
+            <div class="card bg-dark shadow-sm mb-3 text-light">
                 {% include 'core/post-header.html' with post=post %}
-                <div class="card-body border"
-                     style="cursor: pointer"
-                     id="myDiv{{post.id}}" data-url="{% url 'post-detail' post.id %}"
-                >
+                <div class="card-body border post-clickable" id="post-{{ post.id }}" data-url="{% url 'post-detail' post.id %}">
```

Comment:
This post clickable thing is not so good...
How will it work w/o JS? Also, open in new tab will not work on it cause it is js. So really not good. 
Suggestion: add a traditional link (a) in the h3 below. 
Also, for post content, you need to strip tags. And I would also wrap it in href. This way, we don't need ".,post-clickable" at all. 
Please change. 
Javascript should be used as sugar, extra effects, not to handle core HTML functionality like linking.